packages/jsxstyle/preact/jsxstyle-preact.tsx

```diff
@@ -0,0 +1,67 @@
+import * as preact from 'preact';
+import { getStyleCache, componentStyles } from 'jsxstyle-utils';
+import { StyleCache } from 'jsxstyle-utils/src/getStyleCache';
+
+import { CSSProperties } from '../cssproperties';
+
+export const cache: StyleCache = getStyleCache();
+
+export interface ComponentPropProps {
+  className?: any;
+  style?: any;
+  [key: string]: any;
+}
+
+export type ComponentProp =
+  | keyof JSX.IntrinsicElements
+  | preact.AnyComponent<any, any>
+  | any;
+
+export interface JsxstyleProps extends CSSProperties {
+  className?: string;
+  component?: keyof JSX.IntrinsicElements | React.ComponentClass | React.SFC;
+  mediaQueries?: { [key: string]: string };
+  props?: { [key: string]: any };
+  style?: string;
+  children: (JSX.Element | JSX.Element[] | string)[];
```

Comment:
Can't children also be `null` or `undefined` or `false` or an array of any of the above? Does `preact` not have a `Children` type you can use?

Reply:
I haven’t spent more than 30 minutes on the preact version so I’m not 10)% sure. I definitely hope preact has a Children type. I pulled this from a `children` key in preact’s type definition file.